compose/ui/ui-text/build.gradle

```diff
@@ -260,3 +260,7 @@ afterEvaluate {
         t.jvmArgs += ["--add-opens=java.desktop/sun.font=ALL-UNNAMED"]
     }
 }
+dependencies {
```

Comment:
I am not sure about specifying the dependencies here. They are needed to use the `@Immutable` annotation for the `actual` implementations.

@mazunin-v-jb can you please check and let me know if this is the correct place? 🙏🏻

Reply:
It should be already in dependencies, isn't it?